docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -4,19 +4,142 @@
 
 The PagerDuty action type uses the https://v2.developer.pagerduty.com/docs/events-api-v2[v2 Events API] to trigger, acknowledge, and resolve PagerDuty alerts.
 
+* <<pagerduty-benefits, PagerDuty and Elastic integration benefits>>
+* <<pagerduty-connector-configuration, Connector configuration>>
+* <<pagerduty-action-configuration, Action configuration>>
+
+[float]
+[[pagerduty-benefits]]
+=== PagerDuty + Elastic integration benefits
+
+By integrating PagerDuty with Elastic Alerts, you can:
+
+* Route your Elastic Alerts to the right PagerDuty responder within your team, based on your structure, escalation policies, and workflows.
```

Comment:
Route your alerts to the right PagerDuty responder within your team, based on your